src/schemas/SnapsCredential/1-0-0.ts

```diff
@@ -0,0 +1,30 @@
+import { JSONSchema7 } from "json-schema";
+import BaseVerifiableCredential from "../BaseVerifiableCredential/1-0-0";
+
+export default {
+  ...BaseVerifiableCredential,
+
+  $schema: "http://json-schema.org/draft-07/schema#",
+  $id: "https://raw.githubusercontent.com/discoxyz/disco-schemas/main/json/SnapsCredential/1-0-0.json",
+
+  title: "Snaps Credential",
+  description: "Credential containing profile information",
```

Comment:
Is this the right description? I see this in the ticket but not sure if that is meant to be the description:

Send snaps to celebrate a friend or colleague doing something positive and helpful.